platform/jn516x/dev/micromac-radio.c

```diff
@@ -57,14 +57,24 @@
 #include "JPT.h"
 #include "PeripheralRegs.h"
 
-void vMMAC_SetChannelAndPower(uint8 u8Channel, int8 i8power);
-
```

Comment:
I would suggest to update the SDK so the code is inline with the SDK people would download from NXP. I think you have to do it as the SDK is stored on your github.io.